app/src/i18n/locales/de/data.json

```diff
@@ -117,5 +117,14 @@
   "review-data-label": "Daten überprüfen und hinzufügen",
   "files-uploaded": "Dateien hochgeladen",
   "add-custom": "Benutzerdefiniert",
-  "save-missing-scope-info": "Speichern oder fehlende Informationen ausfüllen, bevor Sie fortfahren."
+  "save-missing-scope-info": "Speichern oder fehlende Informationen ausfüllen, bevor Sie fortfahren.",
+  "transform-data-description": "Lorem ipsum dolor sit amet, qui minim labore adipisicing minim sint cillum sint consectetur cupidatat.",
```

Comment:
Please get the actual text for this before we merge the PR

Reply:
I'll ask product for missing text